net-misc/httpie/httpie-2.5.0.ebuild

```diff
@@ -10,11 +10,11 @@ PYTHON_REQ_USE="ssl(+)"
 inherit bash-completion-r1 distutils-r1
 
 DESCRIPTION="Modern command line HTTP client"
-HOMEPAGE="https://httpie.org/ https://pypi.org/project/httpie/"
-SRC_URI="https://github.com/jakubroztocil/httpie/archive/${PV}.tar.gz -> ${P}.tar.gz"
+HOMEPAGE="https://httpie.io/ https://pypi.org/project/httpie/"
+SRC_URI="https://github.com/httpie/httpie/archive/${PV}.tar.gz -> ${P}.tar.gz"
 
 LICENSE="BSD"
-SLOT="0"
+SLOT="1"
```

Comment:
I am maybe missing something, but why is the slot changed?

Reply:
Hm that should not be the case, good catch!